### PR TITLE
Use correct word 'inexistent' -> 'non-existent'

### PR DESCRIPTION
**Pull request by tcharding**
Minor grammar fix in e2e tests.

### CODE REVIEW

**Comment by bonomat** [comment]
@D4nte : you might like this one as you commented exactly that on my PR 😅

File: api_tests/lib_sdk/actors/actor.ts [hunk 1]
```diff
@@ -178,15 +178,15 @@ export class Actor {
 
     public async accept() {
         if (!this.swap) {
-            throw new Error("Cannot accept inexistent swap");
+            throw new Error("Cannot accept non-existent swap");
```
Comment (D4nte) [inline api_tests/lib_sdk/actors/actor.ts]:
```suggestion
            throw new Error("Cannot accept nonexistent swap");
```

File: api_tests/lib_sdk/actors/actor.ts [hunk 2]
```diff
@@ -178,15 +178,15 @@ export class Actor {
...
         }
 
         await this.swap.accept(Actor.defaultActionConfig);
     }
 
     public async fund() {
         if (!this.swap) {
-            throw new Error("Cannot fund inexistent swap");
+            throw new Error("Cannot fund non-existent swap");
```
Comment (D4nte) [inline api_tests/lib_sdk/actors/actor.ts]:
```suggestion
            throw new Error("Cannot fund nonexistent swap");
```

**Comment by tcharding** [comment]
> For pedantry's sake, it looks like inexistent is a [perfectly valid word](https://www.merriam-webster.com/dictionary/inexistent)

Its a word but it has the wrong meaning here.

File: api_tests/lib_sdk/actors/actor.ts (hunk 2 above)
Comment (tcharding) [reply api_tests/lib_sdk/actors/actor.ts]: Cool I learned about hyphens, the web seems to think if it is a 'real' word (which nonexistent is) it does _not_ get hyphens and if it is made up (which I also learned is totally acceptable) it gets hyphens :)

**Comment by tcharding** [comment]
LOL! I misspelled in the commit log the word that this was all about - no `git commit --amend` on github eh?

File: api_tests/lib_sdk/actors/actor.ts [hunk 3]
```diff
@@ -207,7 +207,7 @@ export class Actor {
 
     public async refund() {
         if (!this.swap) {
-            throw new Error("Cannot refund inexistent swap");
+            throw new Error("Cannot refund non-existent swap");
```
Comment (D4nte) [inline api_tests/lib_sdk/actors/actor.ts]: @tcharding maybe stick to one spelling?

**Pull request merged**